src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -173,25 +173,6 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                     throw new RuntimeException("Important information for kitRequest is missing");
                                 }
                             }
-                            //TODO PEGAH GET UNORDERED OR NOT FOUND ORDERS AND ADD THEM  TO THE LIST
-                            //                            addOtherUnorderedKitsToList(kitsToOrder);
-
-                            //only order if kit were added to kits to order hash (which should only be if a kit has an external shipper)
-                            if (!kitsToOrder.isEmpty()) {
```

Comment:
could we remove the whole kitsToOrder map

Reply:
Done--but beware, this cascaded into even more code being deleted :)